### PR TITLE
Task/grav theme updates

### PR DESCRIPTION
**Pull request by jryanconklin**
This PR updates the Grav theme included in Particle: 

* Renamed gravpl => particle for consistency
* Fix JavaScript console error
* Add Bootstrap NavBar Implementation

### CODE REVIEW

File: apps/grav/index.js [hunk 1]
```diff
@@ -13,9 +13,12 @@ console.log('grav-theme ran');
 
 // Send each component the $(document) as its context
 const $context = $(document);
+const settings = {
+  enableHolder: TRUE,
+};
 
 // Let's just execute everything and pass in $(document), settings
 _.forEach(designSystem, (component) => {
```
Comment (illepic) [inline apps/grav/index.js]: @jryanconklin we've got a MUCH more elegant way to do this now, check the drupal/pl index.js files for deets (we now have api methods like `enableAllComponents` on the design-system library!).

File: apps/grav/templates/default.html.twig [hunk 2]
```diff
@@ -1,18 +1,11 @@
...
   {% block content %}
 
     <div>
-      <p>Congratulations! This Grav theme is working! This is a solid proof of concept, but there's a ton more you can do here:</p>
-      <ul>
-        <li>Webpack the Grav</li>
-        <li>Update CSS Assets from Drupal -> Grav</li>
-        <li>Fix JavaScript</li>
-      </ul>
+      <p>Congratulations! This Grav theme is working!
```
Comment (illepic) [inline apps/grav/templates/default.html.twig]: Am I crazy or is there now closing `</p>`?

File: apps/grav/templates/partials/base.html.twig [hunk 3]
```diff
@@ -24,13 +24,17 @@
...
+{% endblock %}
+
 <header class="site__header" role="banner">
   {% block header %}
     {% embed '@molecules/jumbotron/_jumbotron.twig' with { jumbotron_fluid: true } %}
       {% block jumbotron_branding %}
         {% include '@atoms/branding/_branding.twig' with {
-          site_name: 'Particle in ' ~ site.title|e('html'),
-          url: '/',
+          site_name: site.title|e('html'),
```
Comment (illepic) [inline apps/grav/templates/partials/base.html.twig]: What is the `e()` twig filter here?

File: source/_patterns/02-molecules/nav/_nav.twig [hunk 4]
```diff
@@ -32,11 +32,13 @@
...
 
 <{{ nav_element }} class="{{ nav_classes }}" {{ nav_id }} {{ nav_role }}>
-  {% for item in nav_items %}
-    {% if item.nav_item_dropdown %}
-      {% include '@atoms/dropdown/_dropdown.twig' with item %}
-    {% else %}
-      {% include '@molecules/nav/_nav-item.twig' with item %}
-    {% endif %}
-  {% endfor %}
+  {% block nav_items %}
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/_nav.twig]: Verifying that this refactor works back to pl/drupal apps?

File: apps/grav/templates/partials/base.html.twig (hunk 3 above)
Comment (jryanconklin) [reply apps/grav/templates/partials/base.html.twig]: I pulled this from the similar work done in Grav's antimatter theme. My guess was down to the fact that site.title could be set in headmatter on an admin menu. So since that value can accept user input, it should be escaped.

File: source/_patterns/02-molecules/nav/_nav.twig (hunk 4 above)
Comment (jryanconklin) [reply source/_patterns/02-molecules/nav/_nav.twig]: I spun up a Drupal env using this branch to check. Unfortunately I did it post merge of master. There is something wrong with the menu (the admin menu unsets on the page), but it looks like it's an issue with the underscore library as I'm getting a "Uncaught SyntaxError: Invalid or unexpected token" pointing to underscore in browser console.

I don't think it's the inclusion of the block is a problem for Drupal. Specifically because we had to do a bit of custom work to get this working in Drupal's menus such that we're actually not embedding or including `@molecules/nav/_nav.twig` in the Drupal theme. We write out what we need conditionally, then include the nav-items. I think this is okay, but we should open a ticket for underscore.

tl;dr I think this is still okay to merge.

File: apps/grav/templates/partials/base.html.twig (hunk 3 above)
Comment (illepic) [reply apps/grav/templates/partials/base.html.twig]: Ah, `e()` is escape!

File: source/_patterns/02-molecules/nav/_nav.twig (hunk 4 above)
Comment (illepic) [reply source/_patterns/02-molecules/nav/_nav.twig]: That underscore issue really should be solved here: https://github.com/phase2/particle/issues/259

**Comment by illepic** [comment]
@jryanconklin where did the `apps/pl/index.js` changes go?

**Comment by illepic** [comment]
Answered: was already in `master`.
